exporter/debugexporter/README.md

```diff
@@ -0,0 +1,46 @@
+# Logging Exporter
```

Comment:
```suggestion
# Debug Exporter
```

Reply:
Addressed. I've also removed `loglevel` from the readme as the `debug` exporter may allow us to remove `loglevel` altogether